app/services/verifier/acapy_verifier_v1.py

```diff
@@ -1,5 +1,6 @@
 from aries_cloudcontroller import (
     AcaPyClient,
+    ApiException,
```

Comment:
We can revert these changes (and in _v2 service as well). Error handling already improved in incoming PR